LocalMultiplayerAgent/DeploymentTool/DeploymentSettingsValidator.cs

```diff
@@ -0,0 +1,102 @@
+﻿using PlayFab.MultiplayerModels;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Gaming.LocalMultiplayerAgent.DeploymentTool
+{
+    public class DeploymentSettingsValidator
+    {
+        //private readonly ISystemOperations _systemOperations;
+        private readonly DeploymentSettings _settings;
+
+        public DeploymentSettingsValidator(DeploymentSettings settings)
+        {
+            _settings = settings ?? throw new ArgumentException("Deployment settings cannot be null");
+        }
+
+        public bool IsValid()
+        {
+            if (string.IsNullOrWhiteSpace(_settings.BuildName))
```

Comment:
good job on validation check! It would be great if do validation check for VmSize and Region we support. 
For example, We support around 37 Region and 17 VmFamily (with different core option). We can separate file VmSizeInfo.cs and RegionInfo.cs and list out supported Region or VmFamily in there as Enum.

Reply:
Thanks for the hint! I made updates :)